app/_data/tables/support/gateway/packages.yml

```diff
@@ -31,7 +31,7 @@ _packages:
   azurelinux2xx: &azurelinux2xx
```

Comment:
If I understand this correctly, "Azure Linux" support means that Kong was added to the Azure marketplace, which is something that was driven by the partneships team. There should not be an entry in our packages table, so we should remove the whole entry + the entries in the other data files (see original change here: https://github.com/Kong/docs.konghq.com/pull/6721/files) 

We might actually be able to just revert https://github.com/Kong/docs.konghq.com/pull/6721, more accurately.

Reply:
Updating based on our convo that we're going to hold this PR until we've identified new location for the info, tied to a wider conversation about partnership support content. Thanks for great catch and discussion!